<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>951</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="leServerPort">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>30</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>8899</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRestartServer">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>30</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>restart server</string>
    </property>
   </widget>
   <widget class="QListWidget" name="lwdClients">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabClients">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>70</y>
      <width>571</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScanDevice">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>420</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>scan device</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbServiceIp">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="lwdScanResult">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>171</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>951</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
